src/containers/App.tsx

```diff
@@ -79,10 +80,10 @@ class App extends React.Component {
                             <Route exact path="/vote/leaderboard" component={VoterLeaderboard} />
                             <Route exact path="/vote/proposal/:id" component={VoteOverview} />
                             <Route exact path="/vote/address/:address" component={ProposerDetail} />
-                            {process.env.REACT_APP_CHAIN_ID === '97' && (
+                            {CHAIN_ID === BscChainId.TESTNET && (
```

Comment:
maybe it would be useful to add util like `isBscChain`?

Reply:
Yeah good point!
I've added a `isOnTestnet` variable to the config (`isBscChain` would be always true otherwise, since both the mainnet and testnet networks are BSC chains).